### ISSUE TITLE
Add Accessibility Label to Search Autocomplete

### ISSUE THREAD

**Issue by K8Sewell** [yalelibrary/YUL-DC#391]
# Summary
Lighthouse audit of Blacklight site provided area for improvement.  

*  Form elements do not have associated labels - Failing Elements: ```input.search-q.q.form-control.rounded-0.tt-hint```

This issue is coming from the Blacklight base code.  

## Acceptance Criteria
* [ ] PR is up for providing the accessibility label to the base Blacklight repo
* [ ] Release of BlackLight
* [ ] App is upgraded to latest BlackLight release
* [x] Related accessibility tests are not failing

**Comment by K8Sewell** [comment]
Draft PR up on Blacklight
* https://github.com/projectblacklight/blacklight/pull/2318
* Asked for help on Blacklight channel w/  test set up

**Comment by K8Sewell** [comment]
Created additional PR to address issue in yul-dc-blacklight - https://github.com/yalelibrary/yul-dc-blacklight/pull/220
